### DISCUSSION TITLE
. less file support

### DISCUSSION THREAD

**Discussion by xiaoxiji1234** [lowdefy/lowdefy#954] [Questions and Help] [answered]
Hi, Does lowdefy support less style files, or only CSS style files?

**Comment by Gervwyk** [comment, marked as answer]
Hi @xiaoxiji1234 

Unfortunately at the moment no. 

However, this is already something that is part of `v4`. You can track the progress on this on the `v4` branch, but it will still be a month or so before an alpha release of `v4`. Good news is that antd theme variables modifications will also be included in this. So you should be able to customize any of the 100s of antd theme variables 💅
You can view the variable options here.
https://github.com/ant-design/ant-design/blob/master/components/style/themes/default.less

Can you perhaps share a little more detail regarding your use case and how you would like to use `less` files?
